zeppelin-web/src/app/notebook/notebook-actionBar.html

```diff
@@ -67,6 +67,33 @@
               tooltip-placement="bottom" tooltip="Export the notebook">
         <i class="fa fa-download"></i>
       </button>
+        <ul class="dropdown-menu" role="menu" style="width:250px">
+          <li>
+            <div class="cron-preset-container">
+              Git menu
```

Comment:
Can we make this dropdown menu independent from git?
Remove 'git' from Icons, descriptions..